http/server.ts

```diff
@@ -122,6 +122,26 @@ export class ServerRequest {
   conn: Conn;
   r: BufReader;
   w: BufWriter;
+  private _cookie: object;
+
+  get cookie(): object {
+    if (this._cookie) {
+      return this._cookie;
+    }
+    if (this.headers.has("Cookie")) {
+      let out: object = {};
+      const c = this.headers.get("Cookie").split(";");
+      for (let i = 0; i < c.length; i++) {
```

Comment:
```suggestion
      for (const keyValue of c) {
```

Reply:
for of is slower than classic for loops.

Reply:
In general, you shouldn't complicate the code in order to satisfy such micro-optimizations. Even if that's true now, it's likely to not be true in the future. Speed improvements are not won this way, our bottlenecks are many many many orders of magnitude larger.

Reply:
I think the optimization should be given to `v8` to do

Reply:
Using `for loops` instead of `for of` is not really complicating the code. But ok.